### PR TITLE
Add procedure for sync meetings along with agenda template

### PR DESCRIPTION
**Pull request by rylev**
This is the outcome of discussion [here](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Documenting.20Meeting.20Process) and initial drafting [here](https://hackmd.io/@ryanlevick/B1kVrcKD3).

Some callouts:
- This document states that meeting minutes *must* be released at some point. I'm not sure we ever discussed that this should be a requirement, but it seems like the right move. We'll eventually need to figure out the procedure for notes on private agenda items, but we've not yet needed that.
- I did not add a place in the agenda for approving minutes. I would hope that we would not tie approval of minutes to the next meeting. For example, what if the meeting happens 1 month later - is one month an acceptable time for minutes approval? We have discussed the need for good async decision making processes, and I think this decision should use those processes.
- The document states that all meetings are recorded but consensus must be reached on what to do with that recording (including optionally making it public). If no consensus is reached after 1 week, the recording is deleted. There has been discussion of the [pros and cons of recordings in Zulip](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Outputs.20of.20meetings), but consensus has not been reached, and there is major concern about how recording for public consumption could turn the meetings into performative acts (even subconsciously). If folks would prefer to move to a model where consensus must be reached *each* meeting even to record for private use, let me know. My thought was to try to avoid the same conversation each week if it is not necessary to do so. 
- On top of the the hackmd draft I've added the requirement to document a "champion" for each agenda item. Additionally, I've added a meeting norm of calling out in chat when the conversation is getting off topic. 

Checkboxes: 
- [x] @carols10cents 
- [x] @eholk 
- [x] @ehuss 
- [x] @jackh726 
- [x] @khionu 
- [x] @Mark-Simulacrum 
- [x] @m-ou-se 
- [x] @rylev 
- [x] @theJPster

### CODE REVIEW

File: procedures/synchronous-meetings.md
Comment (jonathanpallant) [inline procedures/synchronous-meetings.md:23]:
```suggestion
Detailed notes must be taken of the meeting, even if the meeting was otherwise recorded. The meeting notes must be made public as soon as possible (preferably latest one week after the meeting), but before doing so Council members are asked to review those meeting notes to ensure that the notes reflect their understanding of what occurred in the meeting. Once consensus is reached that the meeting notes are accurate, the notes are released for public viewing. 
```

File: procedures/synchronous-meetings.md
Comment (jonathanpallant) [inline procedures/synchronous-meetings.md:13]: This conflicts with `## Recording`

File: procedures/synchronous-meetings.md
Comment (jonathanpallant) [inline procedures/synchronous-meetings.md:31]:
```suggestion
Recordings are usually used to ensure that the notes are correct and to optionally create automatic transcripts. Recordings can also *optionally* be made public, but only with consensus of all of those present on the recording.
```

File: procedures/synchronous-meetings.md
Comment (jonathanpallant) [inline procedures/synchronous-meetings.md:86]:
```suggestion
- **Points of order**: every participant is encouraged to raise "points of order"  which are meta commentary on some aspect of the meeting that needs clarification or discussion. If a point of order is considered urgent, it may be called out verbally - bypassing the usual queuing mechanism. Such interjections should be as brief as possible, to limit talking over one another. "Point of Order - we are out of our allotted time today", might be a suitable interjection.
```

File: procedures/synchronous-meetings.md
Comment (jonathanpallant) [inline procedures/synchronous-meetings.md:87]:
```suggestion
- **Saying on topic**: while the facilitator is ultimately in charge of keeping the meeting on topic, every participant is encouraged to bring to the facilitator's attention when the conversation seems to have veered off course (either by getting off topic or by becoming too concerned with specific details). A member may wish to raise a Point of Order to highlight this.
```

File: procedures/synchronous-meetings.md
Comment (jonathanpallant) [inline procedures/synchronous-meetings.md:69]: I think this should be an opportunity to have things struck from the agenda, for example if they are not in scope or there are legal issues which might mean a discussion is unwise. I'm not sure it should be an opportunity to add items for discussion as people will not have had time to consider them in advance of the meeting. If we do allow it, the addition of a late item should only be with the consent of the council (c.f. the addition of an item earlier in the process does not require the consent of the council).

File: procedures/synchronous-meetings.md
Comment (ehuss) [inline procedures/synchronous-meetings.md:25]: I have a slight preference for putting them in this repo with something like `minutes/<meeting-name>/YYYY-MM-DD.md`.

File: procedures/synchronous-meetings.md
Comment (ehuss) [inline procedures/synchronous-meetings.md:87]: ?

```suggestion
- **Staying on topic**: while the facilitator is ultimately in charge of keeping the meeting on topic, every participant is encouraged to bring to the facilitator's attention when the conversation seems to have veered off course (either by getting off topic or by becoming too concerned with specific details).
```

**Review by ehuss — COMMENTED** [review]
Thank you for putting this together!

File: templates/sync-meeting-agenda.md
Comment (rylev) [inline templates/sync-meeting-agenda.md:9]: We should also link to any previous discussion on the topic so that folks can get context quickly.

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:25]: I've started a poll on this [here](https://rust-lang.zulipchat.com/#narrow/stream/392734-council/topic/Home.20for.20Sync.20Meeting.20Notes/near/369595603).

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:69]: Ultimately everything (items added earlier in the process as well as last minute additions and deletions) is decided through consent (as the agenda must be consented to by everyone). I've added some notes that try to encourage keeping additions small and not pushing for additions unless consent can be agreed to quickly.

**Review by carols10cents — CHANGES_REQUESTED** [review]
I'd like to see the TODO link to example.com resolved in some manner and the grammar nit fixed before I'm ready to approve. I'm willing to dismiss my comment on calling out the justification of synchronous time if people don't feel it's needed.

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:9]: I'd prefer to have an upper bound on this date as well. Could we say "on the first council meeting on or after July 14, 2023, we will re-evaluate this frequency?"

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:13]: Once this is merged, maybe we should raise an issue to get a little more concrete here. I'm not picky about the software, but it'd be good to have a default.

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:17]: This is probably a different policy question, but have we talked about how we pick alternates? Are alternates a standing position, or is this something the council member can appoint as needed?

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:23]:
```suggestion
Detailed minutes must be taken of the meeting even if the meeting was otherwise recorded. The meeting notes must be made public as soon as possible (preferably at most one week after the meeting), but before doing so Council members are asked to review those meeting notes to ensure that the notes reflect their understanding of what occurred in the meeting. Once consensus is reached that the meeting notes are accurate, the notes are released for public viewing. 
```

Comment (eholk) [inline procedures/synchronous-meetings.md:23]: I'd like to make "as soon as possible" a little more concrete, more to the level of "we promise to release minutes within X time and if not then we have not achieved one of our commitments to the community." I know this may not always be possible, but it'd be nice to be able to produce statistics about how often we meet this commitment. I see making concrete time commitments as a way of building trust with the community because over time we can point to a consistent track record of committing to things and following through.

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:38]: This may be better as a follow-up policy document, but I liked @Manishearth's description of the scribing process they've used in W3C meetings (I've lost the Zulip link for where this was discussed). The idea was to have one person in charge of scribing, other's contribute, and when the scribe is called on to speak there's an explicit delegation to an alternate scribe while the main scribe speaks.

File: procedures/synchronous-meetings.md
Comment (eholk) [reply procedures/synchronous-meetings.md:50]: In meetings I've had with @yaahc we've prefixed items with a tag like (I), (E), (D), which means inform, explore, or decide. I've found this helpful to make it clear at a glance what the purpose of the discussion is.

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:42]: Since currently our meetings haven't been open to the public, I think it'd be good if the agenda document could be public before the meeting so people can know what we're going to be talking about, and also to give council members a chance to get input from their team before the meeting.

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:87]: This says that for urgent points of order they can be made verbally. Is the implication that non-urgent points of order (are their any?) should be raised through the chat instead?

More generally, do we want to add an expectation around use of chat during the meeting? In a lot of calls I'm on we use the chat basically to have side discussions, add clarifying points, or short comments that don't warrant raising a hand. So far we haven't done much of that in council meetings, and I kind of like that. I think it's easier for us all to follow the discussion if we don't have a side stream discussion going.

File: procedures/synchronous-meetings.md
Comment (carols10cents) [reply procedures/synchronous-meetings.md:50]: Oooooh I like that.

File: procedures/synchronous-meetings.md
Comment (ehuss) [reply procedures/synchronous-meetings.md:17]: I believe the RFC indicates that the top-level team is responsible for choosing an alternate ([source](https://rust-lang.github.io/rfcs/3392-leadership-council.html#alternates-and-forgoing-representation)).

So, I'd say the selection process is up to the team. And whether they select ahead-of-time, or on an as-needed basis is also presumably up to the team.

Presumably if a council member knows they are going to be unavailable, they are responsible for letting their team know and get the team to pick someone. If the council member is becomes unavailable without communicating, the council should inform the team.

I'm personally unsure if there should be standing alternates or not. It seems like it might be a good idea just so there isn't a scramble to pick someone. But I could imagine people may not want to be on the hook with uncertainty if they will actually ever be needed.

Comment (rylev) [reply procedures/synchronous-meetings.md:17]: Yes, this was something the governance reform working group explicitly did not want to dictate or have the Council dictate to the top-level teams since it is their responsibility to select representatives how they see fit. However, we *could* offer "best practices" for teams to adopt which they may deviate from if they wish to. Thoughts?

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:38]: This sounds great. I'd like for us to try this out and adopt the process before documenting it as the process though. I'll put this in the agenda for the next meeting to mention that we might want to try something like this.

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:42]: Good point - the agenda's have been public already, but we should document them as such. We should probably figure out standard place to post these so folks can more easily find them. I've stated that the agenda shall be posted as a PR to this repo (which I think makes sense we should have the agenda above meeting minutes).

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:87]: Yea I think having side conversations in chat is not a good idea, but yes, I think we have been using the chat for non-urgent points of order and I think we should continue doing so.

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:57]: I've removed this. We'll need to make a habit of going back to older documents and make sure things are well linked so I don't think this needs an explicit TODO.

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:50]: I like this as well. We can give this a try and see how it sticks though I might avoid usage of abbreviations to make sure the agendas are understandable by those unfamiliar with the procedure.

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md]: I tried to make this more concrete by at least requiring a public announcement when the minutes are delayed. I think that's a good first step. We'll see how this procedure works out in practice, and we can adjust from there.

File: procedures/synchronous-meetings.md
Comment (chorman0773) [inline procedures/synchronous-meetings.md:23]: Two options I've used for meetings that discuss sensitive things is redacting the meeting minutes, or simply having a vote to not release them publically.

File: procedures/synchronous-meetings.md
Comment (chorman0773) [inline procedures/synchronous-meetings.md:42]: This seems to imply that "Facilitator" is a standing role, but then the role is assigned during the metings.
I have resolved such in my groups by having two roles "Chair" and "Speaker" - the Chair being a standing role and the *de facto* default speaker, and handling preparation o the agenda and minutes, and the Speaker being chosen during the meeting to run that meeting. Prior to a Speeker being chosen, the Chair conducts the inital items if present otherwise if they expect to be Absent, they can appoint an acting chair to conduct as much of the Chair's duties as necessary (else, the council could appoint them, perhaps synchronously).

**Review by chorman0773 — COMMENTED** [review]
I've mentioned a few suggested changes in comments.

File: procedures/synchronous-meetings.md
Comment (khionu) [reply procedures/synchronous-meetings.md:17]: If it's meant to be a softer recommendation, I'd prefer to not have it firmly documented, as what we put here will carry weight that may be incorrectly used.

File: procedures/synchronous-meetings.md
Comment (khionu) [inline procedures/synchronous-meetings.md:31]: Nit: s/consensus/consent/.

**Review by carols10cents — APPROVED** [review]
Thank you! I'm good with this now-- please consider my checkmark durable through minor tweaks.

File: procedures/synchronous-meetings.md
Comment (eholk) [reply procedures/synchronous-meetings.md:17]: I agree with the idea of leaving things up to the team where possible. Having some suggestions, like "we recommend teams do X but teams are free to make their own choices based on their needs" would be okay with me, but I think it makes sense to hold off on making these recommendations for now.

File: procedures/synchronous-meetings.md
Comment (eholk) [inline procedures/synchronous-meetings.md:23]:
```suggestion
Detailed minutes must be taken of the meeting even if the meeting was otherwise recorded. The meeting minutes must be made public within one week after the meeting concludes (or a delay must be publicly announced and resolved as soon as possible). After the meeting, Council members must review the minutes to ensure that the minutes reflect their understanding of what occurred in the meeting. Once there is consent that the minutes are accurate, they are released for public viewing.
```

File: procedures/synchronous-meetings.md
Comment (Mark-Simulacrum) [inline procedures/synchronous-meetings.md:53]: As briefly discussed today - some method of pre-raising concerns might be worth adding to agenda to identify stakeholders and topics that need discussion.

File: templates/sync-meeting-agenda.md
Comment (Mark-Simulacrum) [inline templates/sync-meeting-agenda.md:11]:
```suggestion
- Call for clarifications
- Turn off recording
```

**Comment by jonathanpallant** [comment]
Can someone please @ me when all the outstanding comments are resolved?

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:17]: Going to resolve this as it's not really relevant to this document. Happy to see this logged somewhere else though.

File: procedures/synchronous-meetings.md
Comment (rylev) [reply procedures/synchronous-meetings.md:42]: This is a good suggestion. I'm currently trying to document the status quo, but I like this suggestion and will see I can incorporate it into how we do meetings. Currently I've been playing the role of both Chair and Speaker in this scenario.

**Comment by rylev** [comment]
@m-ou-se @khionu @jonathanpallant - this is ready for your review and ✅

**Comment by jonathanpallant** [comment]
Hey cool, TIL you can tick other people's boxes (I just ticked @thejpster's box)

**Review by ehuss — APPROVED** [review]
Considering everyone has checked their box, I'm going to go ahead and merge. Thanks @rylev!

**Pull request merged**
